mappings/net/minecraft/datafixers/fixes/ItemOmniousBannerTranslataionKeyRenameFix.mapping

```diff
@@ -0,0 +1 @@
+CLASS aae net/minecraft/datafixers/fixes/ItemOmniousBannerTranslataionKeyRenameFix
```

Comment:
Ominous?

Reply:
Also, Translation?

Reply:
The key of the key value pair in the translation file was changed. Since the item is technically just a normal banner, the translation key is stored in `tag.display.Name`

Reply:
I really should read my namings again to avoid stupid typos like these.